bigchaindb/tendermint/lib.py

```diff
@@ -16,15 +16,16 @@
 
 import requests
 
-from bigchaindb import backend
-from bigchaindb import Bigchain
+import bigchaindb
```

Comment:
Do we need this import?

Reply:
It was like that in the old `Bigchain` class, and it causes my local unit tests to fail if I change it. I agree that it's ugly, and we should fix it, but I think it's outside the scope of this PR.